OpenRA.Mods.Common/Traits/Player/ProvidesTechPrerequisite.cs

```diff
@@ -40,7 +48,8 @@ public IEnumerable<string> ProvidesPrerequisites
 		public ProvidesTechPrerequisite(ProvidesTechPrerequisiteInfo info, ActorInitializer init)
 		{
 			this.info = info;
-			enabled = info.Name == init.World.LobbyInfo.GlobalSettings.TechLevel;
+			var mapOptions = init.World.WorldActor.TraitOrDefault<MapOptions>();
+			enabled = mapOptions != null && mapOptions.TechLevel == info.Name;
```

Comment:
`info.Id` (sometimes I think you always introduce one stupid error in your PRs just to see if people read them :p)